src/Program.fs

```diff
@@ -422,19 +446,18 @@ let generate (configFile: string) =
                     generatedModules
                     |> Seq.map (fun (path, name, hasVars) -> CodeGen.sampleClientMember (File.ReadAllText(path)) name hasVars)
                     |> String.concat "\n"
-                let clientContent = CodeGen.sampleGraphqlClient config.project clientName config.errorType.typeName members
+                let clientContent = CodeGen.sampleFableGraphqlClient config.project clientName config.errorType.typeName members config.serializer
                 write graphqlClientPath clientContent None
                 colorprintfn "✏️  Generating Fable $green[%s]" projPath
-
-                let files = [
-                    for file in generatedFiles do
-                        createCompileXElement config.createProjectFile outputDirectoryName file
-                ]
-
+                let files =
+                    generatedFiles
+                    |> Seq.map
+                        (fun file -> createCompileXElement config.createProjectFile outputDirectoryName file)
                 let packageReferences = [
                     yield! packageReferences
                     yield MSBuildXElement.PackageReferenceInclude("Fable.SimpleHttp", "3.0.0")
                     yield MSBuildXElement.PackageReferenceInclude("Fable.SimpleJson", "3.21.0")
+                    yield MSBuildXElement.PackageReferenceInclude("Newtonsoft.Json", "12.0.2")
```

Comment:
Fable projects do not work with Newtonsoft.Json, Fable won't be able to compile this project. Please remove `Newtonsoft.Json` from the package references

Reply:
@Zaid-Ajaj there are references to `Newtonsoft.Json` and `Newtonsoft.Json.Linq` in `GraphqlClient` which opened access to `JsonSerializerSettings`, `JsonConverter` etc.

Reply:
No, not for the Fable targets. These don't rely on Newtonsoft.Json

Reply:
Why integration tests fail then?

Reply:
Which integration tests you mean?